scenarios/kubernetes_bazel.py

```diff
@@ -71,10 +71,15 @@ def main(args):
             res = 1
         else:
             try:
-                check(
-                    'bazel', 'run', '//:push-build', '--',
-                    '%s/%s' % (args.gcs, version)
-                    )
+                gcs_location = '%s/%s' % (args.gcs, version)
+                check('bazel', 'run', '//:push-build', '--', gcs_location)
+                artifacts = '%s/_artifacts' % os.environ['WORKSPACE']
+                try:
+                    os.mkdir(artifacts)
+                except OSError:
+                    pass
+                with open(artifacts+'/build_location.txt', 'w') as fp:
```

Comment:
I would like this but I didn't want to muck with the upload paths. We have previous discussion on bazel having a different path from docker builds for some reason.